pkgs/development/python-modules/psycopg/default.nix

```diff
@@ -74,11 +74,17 @@ let
 
     nativeBuildInputs = [
       cython
+      # needed to find pg_config with strictDeps
+      # TODO: switch to pkg-config upstream
```

Comment:
For whom is that TODO? I'm not going to do it, and I don't know whether upstream intends to.